index.js

```diff
@@ -50,6 +49,27 @@ module.exports = {
       fs.unlink(path.join(testAppPath, '.travis.yml')),
     ];
 
+    /**
+     * Setup root package.json scripts based on the packager
+     */
+    tasks.push(
+      (async () => {
+        let packageJson = path.join(options.target, 'package.json');
+        let json = await fs.readJSON(packageJson);
+
+        json.scripts = scripts(options);
```

Comment:
We are overriding the "template-based" scripts section now AFAICT? So the *actual* lint scripts like `lint:js` and `lint:hbs` are missing now, aren't they?

I think we have to decide if we want to split defining the scripts setup using both this programmatic approach and the blueprint "template", or just the former. In the first case, we would need to merge them here (and probably sort by key), in the latter we need to define *all* scripts in the `root-package-json.js` helper and remove them from the template (they are still in there, albeit unused).

Given tests are passing here, it also shows our testing approach is not complete yet. Beside these "smoke tests" we currently have (which run e.g. the `lint` script, which is now passing without actually linting anything), we probably need also some more traditional unit/integration like tests similar to what other blueprints do, testing the output against expected fixtures. Not in this PR of course, just mentioning here...

Reply:
> So the actual lint scripts like lint:js and lint:hbs are missing now, aren't they?

these already don't exist in the root package.json -- I'm matching pairity with the existing `files/package.json`: https://github.com/embroider-build/addon-blueprint/blob/main/files/package.json#L16

> , it also shows our testing approach is not complete yet. 

yup! lots more to do

> testing the output against expected fixtures. Not in this PR of course, just mentioning here...

yeah, I can look in to doing this in a followup -- these are "snapshot tests with extra steps", which help catch accidental / unexpected changes -- which I think we're now ready for testing against

Reply:
ah also, until package.json .. or node in general supports jsonc, the templating we use in blueprints will not-so-great at documenting our context / thoughts / "why" / etc in the blueprint files.